src/bandwidth-tool/templates/picker.vue

```diff
@@ -18,29 +18,33 @@ limitations under the License.
     <div class="picker">
         <div class="tabs">
             <ul>
-                <li v-for="(typeVal, typeKey) in dropletTypes" :class="typeKey === dropletType ? 'is-active' : ''">
+                <li v-for="(typeVal, typeKey) in availableTypes" :class="typeKey === selectedType ? 'is-active' : ''">
                     <a @click="setDropletType(typeKey)">{{ typeVal }}</a>
                 </li>
             </ul>
         </div>
 
-        <!--<div class="switch">
-            <span>{{ i18n.templates.picker.droplets }}</span>
-            <PrettyCheck class="p-switch p-fill" :checked="false" @change="toggleType"></PrettyCheck>
-            <span>{{ i18n.templates.picker.kubernetes }}</span>
+        <!--<div class="variants">
+            <div class="switch">
+                <span>{{ i18n.templates.picker.droplets }}</span>
+                <PrettyCheck class="p-switch p-fill" :checked="type === 'kubernetes'" @change="toggleType"></PrettyCheck>
+                <span>{{ i18n.templates.picker.kubernetes }}</span>
+            </div>
         </div>-->
```

Comment:
Can we leave a comment on why this code is commented out or remove it if its never going to be used?

Reply:
Uncommented this and put it behind a v-if, so it's a bit more apparent when/where this would be visible